tests/rest/devices.spec.js

```diff
@@ -0,0 +1,211 @@
+const chai = require('chai');
+const chai_http = require('chai-http');
+const should = chai.should();
+var server = require('./server-if');
+
+chai.use(chai_http);
+
+describe('Devices interface', () => {
+
+  before(() => {
+    server.start();
+  });
+
+  after(() => {
+  });
+
+  describe('POST /devices', function() {
+    it('should return 201', (done) => {
+      chai.request(server)
+        .post('/devices')
+        .set('Content-Type', 'application/json')
+        .send('{"psk":"cHNrMw==","psk_id":"cHNraWQz","uuid":"GHI"}')
+        .end( (err, res) => {
+          should.not.exist(err);
+          res.should.have.status(201);
+          done();
+        });
+    });
+
+    it('with empty payload should return 400', (done) => {
```

Comment:
In Node.js testing it's common to name tests in certain syntax: describe X -> it 'should Y when Z', so each test describe a situation, i.e. 'server should not crash', 'POST /device should return 400 for empty input', 'bear should roar when poked', etc.